packages/components/src/components/primitives/form/DropdownField/DropdownNativeSelect/DropdownNativeSelect.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { forwardRef, useState } from 'react';
+import cn from 'classnames';
+import { Icon } from 'components/primitives';
+import { expand_more as ExpandMore } from 'components/primitives/Icon/icons/navigation';
+import { getDropdownSingleSelectClasses } from 'utils/getDropdownSingleSelectClasses';
+
+export const DROPDOWN_NATIVE_DEFAULT_TEST_ID = 'dropdown_native';
+
+export interface Option {
+  label: any;
+  value: any;
+}
+
+type Props = {
+  className?: string;
+  options?: Option[];
+  placeHolder?: string;
+  initialSelectedOption?: Option;
+  onItemChange?: (item: Option) => void;
+  testId?: string;
+  disabled?: boolean;
+  name?: string;
+  invalid?: boolean;
+};
+
+export const DropdownNativeSelect = forwardRef<HTMLSelectElement, Props>(
+  (
+    {
+      className,
+      options,
+      placeHolder = 'Select an item',
+      initialSelectedOption,
+      onItemChange,
+      testId = DROPDOWN_NATIVE_DEFAULT_TEST_ID,
+      disabled = false,
+      name = 'dropdown',
+      invalid,
+    }: Props,
+    ref
+  ) => {
+    const defaultValue = initialSelectedOption && initialSelectedOption.value;
+    const [itemSelected, setItemSelected] = useState(defaultValue || '');
```

Comment:
this is assuming that `defaultValue` is a string but the definition of the Option is of type `any`

Reply:
For now, I change the label to a string, we need to validate with the design team wich values could take this dropdown, for example, I saw a list with icon and label.